Libplanet/Net/Protocols/RoutingTable.cs

```diff
@@ -7,6 +7,9 @@ namespace Libplanet.Net.Protocols
 {
     internal class RoutingTable
     {
+        // FIXME: This would be configurable.
+        private const int _minPeersToBroadcast = 8;
```

Comment:
The minimum number of peers to broadcast in the description is 10. Is this okay?

Reply:
Oh, I'll fix it :(